public-docs/shipping-admin.md

```diff
@@ -28,346 +24,4 @@ Confused? See [Concepts: Fulfillment](concepts-fulfillment.md)
     - Rate: The shipping price to charge for this shipping method at checkout.
     - Enabled: Check this to enable this option.
 
-3. Click **Save Changes**.
-
-## Shipping Restrictions
```

Comment:
I think we should keep these `Shipping Restrictions` docs _somewhere_, even though we don't have Store Operator-level UI for it yet. It could go in the Concepts: Fulfillment section - https://docs.reactioncommerce.com/docs/concepts-fulfillment or perhaps another page entirely in Concepts. What do you think @kieckhafer?

Reply:
That seems like a good place for them to be for the time being until we get that UI built (which should be built alongside the upcoming Promotions, if the plan is still the same)